web/CASTNXT/app/controllers/api_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ApiController < ApplicationController
+  # GET /user
+    def index
+        talents = Talent.all
+        render json:talents
+    end
+    
+    def change_status
+        user = Talent.find_by(:_id => params[:id] )
+        user.is_active = !user.is_active
+        user.is_valid = !user.is_valid
```

Comment:
user.is_valid change is not required here, as you guys have asked to only update user.is_active